google-cloud-spanner/src/test/java/com/google/cloud/spanner/ReadWriteTransactionWithInlineBeginTest.java

```diff
@@ -389,6 +404,71 @@ public void failedBatchUpdateAndThenUpdate() {
     assertThat(countTransactionsStarted()).isEqualTo(2);
   }
 
+  @Test
+  public void executeSqlWithOptimisticConcurrencyControl() {
+    client
+        .readWriteTransaction(Options.optimisticLock())
+        .run(
+            transaction -> {
+              try (ResultSet rs = transaction.executeQuery(SELECT1)) {
+                while (rs.next()) {
+                  assertThat(rs.getLong(0)).isEqualTo(1);
```

Comment:
nit (here and elsewhere): prefer the standard org.junit.Assert `assertEquals(expected, actual)` over `assertThat(..)`.

(I know that there's a lot of Google Truth `assertThat(..)` in this and also other files for historical reasons, but the preference for the client libraries is to use `assertEquals` for new code)

Reply:
Done